app/models/concerns/search.rb

```diff
@@ -40,6 +26,17 @@ def status(param)
       param == 'active' ? where(active: true) : where(active: false)
     end
 
+    def keyword(k)
+      tsquery = <<-SQL
+        (to_tsquery('english', ''' ' || '#{k}' || ' ''' || ':*'))
```

Comment:
Actually, I guess you're talking about this one: http://apidock.com/rails/ActiveRecord/Sanitization/ClassMethods/sanitize

Not sure why it's not listed on the previous page.